api/client/events.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/brimdata/zed/api"
+	"github.com/brimdata/zed/zson"
+)
+
+type EventsClient struct {
+	resp        *Response
```

Comment:
Nit: Don't really need the response here, just the body.
```suggestion
	rc        io.ReadCloser
```